### PR TITLE
Add New Port DEVOLVER Bootleg

### PR DESCRIPTION
**Pull request by MadShmupper**
# New Port for { DEVOLVER Bootleg }

## Game Information
- **Title**: { DEVOLVER Bootleg }
- **URL**: { https://store.steampowered.com/app/1066260/Devolver_Bootleg }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by kloptops — APPROVED** [review]

File: ports/devbootleg/DEVOLVER Bootleg.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+export PORT_32BIT="Y"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/devbootleg"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
```
Comment (Cebion) [inline ports/devbootleg/DEVOLVER Bootleg.sh:26]: Wrong ld_Library export, check the example scripts how they.need to look.

File: ports/devbootleg/DEVOLVER Bootleg.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+export GMLOADER_PLATFORM="os_linux"
+
+# We log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFW_NAME}.txt" ]; then
```
Comment (Cebion) [inline ports/devbootleg/DEVOLVER Bootleg.sh:36]: Gmloader does not have libgl / gl4es so this is not needed.

File: ports/devbootleg/gameinfo.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./devbootleg</path>
```
Comment (Cebion) [inline ports/devbootleg/gameinfo.xml:4]: Path is wrong

**Review by Cebion — DISMISSED** [review]

File: ports/devbootleg/DEVOLVER Bootleg.sh (hunk 1 above)
Comment (JeodC) [inline ports/devbootleg/DEVOLVER Bootleg.sh:26]:
```bash
export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"
```

File: ports/devbootleg/DEVOLVER Bootleg.sh (hunk 2 above)
Comment (JeodC) [inline ports/devbootleg/DEVOLVER Bootleg.sh:36]: Not needed

File: ports/devbootleg/DEVOLVER Bootleg.sh [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFW_NAME}.txt" ]; then 
+  source "${controlfolder}/libgl_${CFW_NAME}.txt"
+else
+  source "${controlfolder}/libgl_default.txt"
+fi
+
+# Rename data.win to game.droid if it exists in ./gamedata
```
Comment (JeodC) [inline ports/devbootleg/DEVOLVER Bootleg.sh:42]: Account for game.unx if there's a linux version?

File: ports/devbootleg/gameinfo.xml (hunk 3 above)
Comment (JeodC) [inline ports/devbootleg/gameinfo.xml:4]: This should be the sh filename

./DEVOLVER Bootleg.sh

**Review by JeodC — CHANGES_REQUESTED** [review]

File: ports/devbootleg/DEVOLVER Bootleg.sh (hunk 4 above)
Comment (JanTrueno) [reply ports/devbootleg/DEVOLVER Bootleg.sh:42]: There is no linux version of the game so no need to rename game.unx, doesnt exist. This line is correct :)

**Review by Slayer366 — APPROVED** [review]

**Review by JeodC — APPROVED** [review]

**Pull request merged**